### ISSUE TITLE
Converting tau when data is log transform

### ISSUE THREAD

**Issue by ciberger** [MasaAsami/pysynthdid#2]
Hey! thanks for your package 

I've been playing around with it as an extension of Google's CausalImpact tool.

Got a dataset with market data having different scales, so log transformation is needed. However, I couldn't transform back ATT from its value estimated from log input data to one unscaled. Which formula should I use? Also, is it possible to get confidence interval for tau?

**Comment by MasaAsami** [comment]
Hi! Thank you for your comment.

>  However, I couldn't transform back ATT from its value estimated from log input data to one unscaled.

As a simple example, consider a linear model. Think about the meaning of the following x1 coefficient when the outcome variable Y is log-transformed.

<img width="249" alt="image" src="https://user-images.githubusercontent.com/16971400/168430188-f739881c-9edb-44e4-8064-5b494302a0e5.png">

As you can see from this

β1 means "Y increases by β1( = log Y'- log Y) when x1 increases by 1".

Since log(1 + y) can be approximated as y, the following transformation is possible:
log Y'- log Y = log(Y'/Y)
　　　　　　= log(1 + (Y' - Y)/Y )
　　　　　　 ~ (Y' - Y) / Y

In other words, we can think of it as "Y increases by β% as x1 increases by 1.
This is the same in this model, which can be interpreted as an ATT of {tau}%!

I apologize if I misread the intent of your question.

**Comment by samsunknight** [comment]
Hi, thanks so much for this package! I'm really grateful to you for coding this up and building it. For one of my applications, however, I'm finding some apparent discrepancies between the graphed quantities and the reported $\tau$. For example, see the below screenshot:

<img width="697" alt="Screen Shot 2023-03-07 at 6 04 53 PM" src="https://user-images.githubusercontent.com/42912309/223574236-a5663599-7cb7-4ead-9823-fe44b5e75665.png">

This $\tau$ seems obviously much larger than the visible difference between the graphed lines, which inverts from roughly 0.1 above to 0.1 below, suggesting a $\tau$ closer to -0.2. 

Can you help me understand what might be causing this discrepancy? I was trying to debug by inspecting the fixed effects, but I wasn't able to see in the code where the fixed effects are stored, so had trouble understanding what might have gone amiss; I'm not sure if the plot is wrong, if $\tau$ is wrong, or if there's just something odd about the relatively sparse $\lambda$ weighting that is leading to the mismatch between the estimate and the visual pattern. I'd be happy to provide replication data if that would be helpful, thanks regardless for your attention to this.
